imports/ui/QuestionSidebar.jsx

```diff
@@ -173,25 +176,25 @@ export class QuestionSidebar extends ControlledForm {
     })
   }
 
-  resetFilter(){
+  resetFilter () {
```

Comment:
I also find attaching the () to the name of the function is clearer, but I'm not an expert in JS styling... so whatever it's supposed to be....

Reply:
I would agree as well. This may have been accidental. I'll update this.

Reply:
Actually, this is a standard.js rule; `Space after function name function name (arg) { ... }' https://standardjs.com

If we _really_ wanted to, we could use something more configurable like eslint if we really wanted to, but the idea with Standard is that's its a bunch of "sane defaults" so that we don't really have to worry about writing the rules - just to follow them. As they say:

"The whole point of standard is to save you time by avoiding bikeshedding about code style. There are lots of debates online about tabs vs. spaces, etc. that will never be resolved. These debates just distract from getting stuff done. At the end of the day you have to 'just pick something', and that's the whole philosophy of standard -- its a bunch of sensible 'just pick something' opinions. Hopefully, users see the value in that over defending their own opinions."